tasks/__init__.py

```diff
@@ -300,11 +301,16 @@ def test_module(ctx, module=None, numprocesses=None, nocapture=False, params=Non
         args += ['-n {}'.format(numprocesses), '--max-slave-restart=0']
     modules = [module] if isinstance(module, basestring) else module
     args.extend(modules)
+    if testmon:
+        args.extend(['--testmon'])
+
     if params:
         params = [params] if isinstance(params, basestring) else params
         args.extend(params)
     retcode = pytest.main(args)
-    sys.exit(retcode)
+
+    # exit code 5 is all tests skipped which is the same as passing with testmon
+    sys.exit(0 if retcode == 5 else retcode)
```

Comment:
Could we make a constant at the top of the file such as `NO_TESTS_COLLECTED = 5` so this could be `if retcode == NO_TESTS_COLLECTED`? This would avoid magic numbers.